wiki/Beatmap_Editor/AiMod/es.md

```diff
@@ -1,165 +1,176 @@
+- **All:** Junta los ítems de todas las demás pestañas.
+- **Compose:** Estos problemas son generalmente por el posicionamiento de objetos. Quizá debas de marcar la caja de "Check distance snap" para que estas alertas aparezcan (esta acción podría causar lag, dependiendo de la duración/tamaño del beatmap).
+- **Design:** Estos problemas son generalmente causados por problemas de la interface, tales como imagen de fondo, storyboards, etc.
+- **Timing:** Cuando estos problemas aparecen, revisa la línea de tiempo porque algo está probablemente fuera de lugar.
+- **Meta:** Cuando estos problemas aparecer, revisa los ajustes de la canción.
+- **Mapset:** Estos problemas afectan al set de mapas en su totalidad.
 
 ## Mensajes
 
-> Nota: los números dentro de los corchetes (es decir, "{0}") son un marcador de posición para un número o una palabra.
+*Nota: Los números dentro de llaves (ej. `{0}`) son marcadores para un número o una palabra.*
 
 ### Informativo
 
 #### All
 
-| Mensaje | Explicación| Solución |
+| Mensaje | Explicación | Solución |
 | :-- | :-- | :-- |
-| ¡No se encontraron problemas en este mapa! | ¡AiMod no pudo encontrar ningun problema con tu mapa! | Podrias subir tu beatmap usando BSS para permitir que los osu!modders encuentren otros errores y problemas con tu beatmap. |
+| No problems were found in this map! | ¡AiMod no pudo encontrar ningún problema con tu mapa! | Puedes subir tu beatmap usando BSS para permitir a los osu!modders encontrar algún otro error o problema en tu beatmap. |
 
 #### Meta
 
 | Mensaje | Explicación | Solución |
 | :-- | :-- | :-- |
-| Se recomienda que la tasa de HP para Fácil/Normal sea como mínimo 4. | ¡Mapas de osu!mania solamente! | Ve a "Song Settings" y configura la velocidad de HP en 4 o superior |
-| Se recomienda que la tasa de HP para Difícil y superior sea de al menos 7. | ¡Mapas de osu!mania solamente! | Ve a "Song Settings" y configura la frecuencia de HP en 7 o superior |
-| Se sugiere que la tasa de OD para mapas con pocos sliders sea al menos 7. | ¡Mapas de osu!mania solamente! | Ve a "Song Settings" y configura la tasa de OD en 7 o más |
-| La velocidad del control deslizante debe ser 1.40 o 1.60. | ¡Mapas de osu!taiko solamente! | Ve a la pestaña "Timing" y cambia la Velocidad de los sliders a 1.40 o 1.60 |
+| HP rate for Easy/Normal is suggested to be at least 4. | ¡Solo en mapas de osu!mania! | Ve a las configuraciones de la canción y fija el HP rate a 4 o mayor |
+| HP rate for Hard and up is suggested to be at least 7. | ¡Solo en mapas de osu!mania! | Ve a las configuraciones de la canción y fija el HP rate a 7 o mayor |
+| OD rate for maps with few sliders is suggested to be at least 7. | ¡Solo en mapas de osu!mania! | Ve a las configuraciones de la canción y fija el OD rate a 7 o mayor |
+| The Slider Velicity should be 1.40 or 1.60. | ¡sólo en mapas de osu!taiko! | Ve a la ventana de Timing y cambia la Velocidad del Deslizador a 1.40 o 1.60 |
```

Comment:
I just realized that I didn't translate the message on this table like in the others, should I?

Reply:
```suggestion
| HP rate for Easy/Normal is suggested to be at least 4. | ¡Solo en mapas de osu!mania! Se recomienda que la tasa de HP para Easy/Normal sea como mínimo 4. | Ve a las configuraciones de la canción y fija el HP rate a 4 o mayor |
| HP rate for Hard and up is suggested to be at least 7. | ¡Solo en mapas de osu!mania! Se recomienda que la tasa de HP para Hard o más difíciles sea como mínimo 7. | Ve a las configuraciones de la canción y fija el HP rate a 7 o mayor |
| OD rate for maps with few sliders is suggested to be at least 7. | ¡Solo en mapas de osu!mania! Para mapas con pocos deslizadores se recomienda una tasa de OD como mínimo 7. | Ve a las configuraciones de la canción y fija el OD rate a 7 o mayor |
| The Slider Velicity should be 1.40 or 1.60. | ¡sólo en mapas de osu!taiko! La velocidad de los deslizadores debe estar entre 1.40 y 1.60. | Ve a la ventana de Timing y cambia la Velocidad del Deslizador a 1.40 o 1.60 |
```